client/rpcserver/handlers.go

```diff
@@ -13,6 +13,7 @@ import (
 	"decred.org/dcrdex/client/core"
 	"decred.org/dcrdex/dex"
 	"decred.org/dcrdex/dex/encode"
+	"decred.org/dcrdex/dex/encrypt"
```

Comment:
`encode.ClearBytes` should suffice in this fine too

Reply:
Ok.